pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -12,28 +16,26 @@ let
       license ? lib.licenses.mit,
       maintainers ? [ lib.maintainers.phanirithvij ],
     }:
-    fetchurl {
-      inherit
-        hash
-        url
-        pname
-        version
-        ;
-      name = "${pname}-${version}.wasm";
+    stdenv.mkDerivation (finalAttrs: {
+      inherit pname version;
+      src = fetchurl { inherit url sha256; };
```

Comment:
Would it be possible to prefer `hash` here instead? I'm concerned that continuing to use `sha256` might introduce troubles in the context of the following related issues. 

https://github.com/NixOS/nixpkgs/issues/325892
https://github.com/NixOS/nixpkgs/pull/389461

Reply:
will do